src/mol-model/structure/model/properties/utils/secondary-structure.validation

```diff
@@ -0,0 +1,73 @@
+compares Mol* port of DSSP (with default parameters) to the BioJava implementation
```

Comment:
would be good to move that into a spec.ts test (once we fixed the issue with jest and circular dependencies)

Reply:
Keeping that in mind for later.